homeassistant/components/unifiprotect/const.py

```diff
@@ -75,3 +77,8 @@
 DISPATCH_ADD = "add_device"
 DISPATCH_ADOPT = "adopt_device"
 DISPATCH_CHANNELS = "new_camera_channels"
+
+EVENT_TYPE_FINGERPRINT_IDENTIFIED: Final = "fingerprint_identified"
+EVENT_TYPE_FINGERPRINT_NOT_IDENTIFIED: Final = "fingerprint_not_identified"
+EVENT_TYPE_NFC_SCANNED: Final = "nfc_card_scanned"
```

Comment:
```suggestion
EVENT_TYPE_NFC_SCANNED: Final = "scanned"
```

I think we can drop `nfc_card` since it doesn't have to be a card